.changeset/better-socks-stand.md

```diff
@@ -0,0 +1,5 @@
+---
+'@noaignite/react-utils': minor
+---
+
+fix stable references of callbacks with stale values
```

Comment:
This changeset might need two separate entries. The version change describes a `minor` whilst the description describes a patch. This should ideally be:

Changeset 1
```
---
'@noaignite/react-utils': minor
---

Add new `useStableCallback` hook
```

Changeset 2
```
---
'@noaignite/react-utils': patch
---

Fix stable references of callbacks with stale values
```

I have not tried this myself but perhaps that this could be a single changeset like below:
```
---
'@noaignite/react-utils': minor
---

Add new `useStableCallback` hook
Fix stable references of callbacks with stale values
```